packages/manager/src/views/WebPropertyDetailPage/components/workflow3.0/ConfigureWorkflowForm.tsx

```diff
@@ -119,11 +127,14 @@ export const ConfigureWorkflowForm = ({
   } = useForm<FormData>({
     defaultValues: {
       ...dataProps,
-
-      config: keyValuePairsGenerator({ dataProps, property: 'config' }).map((item) => ({
-        key: item.key,
-        value: item.value as string | undefined
-      })),
+      config: keyValuePairsGenerator({
+        dataProps: dataProps.config,
+        property: 'spashipWorkflowSecret'
+      }).concat(
+        Object.entries(dataProps.config)
+          .filter(([key]) => key !== 'spashipWorkflowSecret')
```

Comment:
`spashipWorkflowSecret` use in a variable as it's being used multiple places

Reply:
Fixed